packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -403,7 +403,7 @@ function createProjectionAndParams({
                     }
                 }
 
-                res.projection.push(`${field.alias}: ${field.alias}${offsetLimitStr}`);
+                res.projection.push(`${field.alias}: ${field.name || field.alias}${offsetLimitStr}`);
```

Comment:
This is interesting, because `field.alias` defaults to `field.name` if an alias has not been set! 😕

Reply:
Actually, let's think about this. As far as I'm aware, `field.name` is _always_ populated - so the `||` here will never be reached - it might be worthwhile looking into this a bit more.

Reply:
Yes, good point. Let me have a closer look at this.

Reply:
You're right @darrellwarde in regards to `field.alias` defaulting to `field.name` if there is no alias specified!
I dropped the fallback to `field.alias` in https://github.com/neo4j/graphql/pull/1556/commits/5d5a0def5cf8a9d5577ebe43f1456372b4ddf723.
Reason: In the case of the result projection, we always use the `field.name` as the _value_ since we use the `field.name` as the variable name in the foregoing `CALL subqueries`, see the `events` variable in the [tck test](https://github.com/neo4j/graphql/pull/1556/files#diff-4d3c520cf33cd25706cdc78896581b8e1bd9985be6ebc8dd73e947f9144f7af1R90-R96).